docs/template/public/less/_guidelines.less

```diff
@@ -0,0 +1,301 @@
+@import '_variables'; 
+
+// Patterns page styles
+
+.kss-guidelines-page {
+    margin-bottom:50px;
+
+    .imageCenter {
+        position: relative;
+        margin-left: auto;
+        margin-right: auto;
+        text-align: center;
+        margin-top: -40px;
+    }
+
+    a img, a span {
+        vertical-align: middle;
+    }
+
+    #hyGuidelinesContianer {
+        padding-bottom: 20px;
+    }
+
+    .carousel-inner > .item > img {
+        margin: 0 auto;
+    }
+
+    .visualSystemSpacing {
+        padding-top: 40px;
+    }
+}
+
+.visual-system {
+
+    /* font-size */
+    .size-26 {
+        font-size: 26px;
+    }
+
+    .size-20 {
+        font-size: 20px;
+    }
+
+    /* 5 column layout */
+    .col-xs-5ths,
```

Comment:
I don't think we're using this anymore. My search shows it's only in the less, so we should remove it